addons/binding/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/handler/CBusGroupHandler.java

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
could you please re-run "mvn license:format" to update the license headers to the current version?